sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.aest.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Controller, Param, ParseIntPipe, Patch } from "@nestjs/common";
+import {
+  ApiNotFoundResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import {
+  AllowAuthorizedParty,
+  Groups,
+  Roles,
+  UserToken,
+} from "../../auth/decorators";
+import { Role } from "../../auth/roles.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { IUserToken } from "../../auth/userToken.interface";
+import { COE_NOT_FOUND_MESSAGE } from "../../constants";
+import { ClientTypeBaseRoute } from "../../types";
+import { COE_WINDOW } from "../../utilities";
+import BaseController from "../BaseController";
+import { ConfirmationOfEnrollmentControllerService } from "./confirmation-of-enrollment.controller.service";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("confirmation-of-enrollment")
+@ApiTags(`${ClientTypeBaseRoute.AEST}[Confirmation Of Enrollment - CoE]`)
```

Comment:
we were following this format in all other controllers, 
![image](https://user-images.githubusercontent.com/77353155/211071148-501cd8a2-c81a-40af-8ba8-4ba023ccda16.png)

Reply:
@ann-aot, I believe that @dheepak-aot was trying to follow the below one that we have for locations.
```ts
@ApiTags(`${ClientTypeBaseRoute.Institution}-location[Confirmation Of Enrollment - CoE]`)
```
For the locations, it was added to create the idea of a subitem of the location.
I do not see this as the same for the AEST. I would recommend to use the below.
```ts
@ApiTags(`${ClientTypeBaseRoute.Institution}-Confirmation Of Enrollment - CoE`,)
```
or
```ts
@ApiTags(`${ClientTypeBaseRoute.Institution}-confirmation-of-enrollment`,)
```
and change the location one also to
```ts
@ApiTags(`${ClientTypeBaseRoute.Institution}-location[confirmation-of-enrollment]`)
```